src/nvim/menu.c

```diff
@@ -1156,14 +1154,18 @@ static char_u *popup_mode_name(char_u *name, int idx)
 }
 
 
-/*
- * Duplicate the menu item text and then process to see if a mnemonic key
- * and/or accelerator text has been identified.
- * Returns a pointer to allocated memory, or NULL for failure.
- * If mnemonic != NULL, *mnemonic is set to the character after the first '&'.
- * If actext != NULL, *actext is set to the text after the first TAB.
- */
+/// Duplicate the menu item text and then process to see if a mnemonic key
+/// and/or accelerator text has been identified.
+///
+/// @param str
```

Comment:
I agree with all your points, I'll give more attention to Doxygen comments next time.
The updated PR is on the way.